### PR TITLE
Feature/address component style

### PR DESCRIPTION
**Pull request by auredufour**
- Add Css for the set location
- change the mark-up on the Address window and AddressListItems

### CODE REVIEW

**Comment by cgolinski** [comment]
Styling looks great!

File: src/styles/App/_address-modal.scss [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  width: 60%;
+  margin: 0 auto;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+  border-radius: 5px;
+  padding: 50px;
+}
+
+.adress-container {
```
Comment (epq) [inline src/styles/App/_address-modal.scss]: You have a type for "address" here.

**Comment by auredufour** [comment]
@epq you are right for the font, and the typo, I correct both of them!

**Comment by srhboo** [comment]
👍 looks beautiful!

**Pull request merged**